web/src/scheduled_messages.js

```diff
@@ -324,6 +360,10 @@ export function initialize(scheduled_messages_params) {
         e.preventDefault();
         e.stopPropagation();
     });
+
+    $("body").on("focus", ".overlay-message-info-box", (e) => {
+        drafts.activate_element(e.target, keyboard_handling_context);
+    });
```

Comment:
The changes in this file all belong in `scheduled_messages_overlay_ui.js`, since they are implementation used for that overlay only.

Reply:
Moved the changes into `scheduled_messages_overlay_ui.js`!